mgmt_cli_test.py

```diff
@@ -1550,3 +1555,93 @@ def test_restore_data_without_manager(self):
                                "Please provide the 'mgmt_reuse_backup_snapshot_name' parameter.")
 
         self.test_restore_from_precreated_backup(snapshot_name, restore_outside_manager=True)
+
+
+class ReportType(Enum):
```

Comment:
And the same here about class name, please

Reply:
Done